### ISSUE TITLE
postman app login is failing  with problems connecting to  our authentication service

### ISSUE THREAD

**Issue by juergenschubert** [a85/POSTMan-Chrome-Extension#8838]
**Describe the bug**
With the desktop postman  Postman v7.29.1 on WIndows and when I try to login it is telling me:  Something went wrong! we are having problems connecting to  our authentication service

**To Reproduce**
Stat the postman app
1. Go to "Sign IN"
2.use working credential

 Something went wrong! we are having problems connecting to  our authentication service

**Expected behavior**
need a login

here's the console log:
vendor-shared.js:188481 Initializing the global config
vendor-shared.js:249314 App updater event Object
vendor-shared.js:249314 App updater event Object
vendor-shared.js:249314 Disconnected from Interceptor
vendor-shared.js:249314 Use `shouldComponentUpdate` in an `observer` based component breaks the behavior of `observer` and might lead to unexpected results. Manually implementing `sCU` should not be needed when using mobx-react.
(anonymous) @ vendor-shared.js:249314
vendor-shared.js:249314 Warning: a promise was created in a handler but was not returned from it    at Deferred.toPromise (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:359456:19)    at Deferred.then (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:359315:22)    at ConsoleEventStore.updateMeta (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:677588:5)    at file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:677527:14    at transaction$$1 (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:21864:26)    at file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:677525:72From previous event:    at ConsoleEventStore.hydrate (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:677524:5)    at executeAction$$1 (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:19833:19)    at ConsoleEventStore.res (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:19825:16)    at new BaseListStore (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:52217:116)    at new ConsoleEventStore (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:677418:5)    at getStore (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:17360:19)    at new ConsoleStore (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:678007:90)    at getStore (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:17360:19)    at new RequesterBottomPaneHeader (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/RequesterInit.js:22724:93)    at Lg (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353226:171)    at fi (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353268:149)    at Rj (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353355:490)    at Qj (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353338:265)    at Kj (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353338:194)    at yj (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353331:172)    at file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353215:115    at exports.unstable_runWithPriority (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353424:467)    at cg (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353214:325)    at fg (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353215:61)    at gg (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353214:496)    at Ig (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353322:156)    at Object.enqueueSetState (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353224:237)    at LoadableComponent.F.setState (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/vendor-shared.js:353072:369)    at update (file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/requester.js:15548:16)    at file:///C:/Users/Administrator/AppData/Local/Postman/app-7.29.1/resources/app.asar/js/requester.js:15558:9
(anonymous) @ vendor-shared.js:249314
config.getpostman.com/v1/bootstrap:1 Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID
vendor-shared.js:249314 Use `shouldComponentUpdate` in an `observer` based component breaks the behavior of `observer` and might lead to unexpected results. Manually implementing `sCU` should not be needed when using mobx-react.
(anonymous) @ vendor-shared.js:249314
vendor-shared.js:249314 Use `shouldComponentUpdate` in an `observer` based component breaks the behavior of `observer` and might lead to unexpected results. Manually implementing `sCU` should not be needed when using mobx-react.
(anonymous) @ vendor-shared.js:249314
vendor-shared.js:249314 Could not find on CollectionStore. Missing or invalid parameter.
(anonymous) @ vendor-shared.js:249314
vendor-shared.js:249314 Could not find on CollectionStore. Missing or invalid parameter.
(anonymous) @ vendor-shared.js:249314
vendor-shared.js:249314 Could not find on CollectionStore. Missing or invalid parameter.
(anonymous) @ vendor-shared.js:249314
sync-v4.getpostman.com/notification?limit=10&events=viewed&app_id=471842a4-9ed4-4ee7-8a14-611852282c9c:1 Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID
vendor-shared.js:249314 Could not find on CollectionStore. Missing or invalid parameter.
(anonymous) @ vendor-shared.js:249314
analytics.getpostman.com/events:1 Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID
vendor-shared.js:95308 Uncaught (in promise) Objecterror: TypeError: Failed to fetchstatus: 1__proto__: Object
sentry-app.postmanlabs.com/api/22/store/?sentry_key=8fb2d5d1749642eface793d3dd93a6d8&sentry_version=7:1 Failed to load resource: the server responded with a status of 429 ()
analytics.getpostman.com/events:1 Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID
vendor-shared.js:95308 Uncaught (in promise) Object
vendor-shared.js:249314 Could not find on EditorStore. Missing or invalid parameter.
(anonymous) @ vendor-shared.js:249314
vendor-shared.js:249314 Could not find on EditorStore. Missing or invalid parameter.
(anonymous) @ vendor-shared.js:249314
analytics.getpostman.com/events:1 Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID
vendor-shared.js:95308 Uncaught (in promise) Object
analytics.getpostman.com/events:1 Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID
vendor-shared.js:95308 Uncaught (in promise) Object
analytics.getpostman.com/events:1 Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID
vendor-shared.js:95308 Uncaught (in promise) Object
vendor-shared.js:249314 App updater event {name: "checkForElectronUpdate", namespace: "app-update"}
vendor-shared.js:249377 POST https://analytics.getpostman.com/events net::ERR_CERT_AUTHORITY_INVALID
(anonymous) @ vendor-shared.js:249377
request @ vendor-shared.js:95315
sendRequest @ vendor-shared.js:697362
sendPayloads @ vendor-shared.js:697338
(anonymous) @ vendor-shared.js:697181
sentryWrapped @ vendor-shared.js:248801
vendor-shared.js:95308 Uncaught (in promise) {error: TypeError: Failed to fetch, status: 1}
(anonymous) @ vendor-shared.js:95308
Promise.catch (async)
(anonymous) @ vendor-shared.js:95305
wrap @ vendor-shared.js:95278
request @ vendor-shared.js:95315
sendRequest @ vendor-shared.js:697362
sendPayloads @ vendor-shared.js:697338
(anonymous) @ vendor-shared.js:697181
sentryWrapped @ vendor-shared.js:248801
vendor-shared.js:249377 POST https://analytics.getpostman.com/events net::ERR_CERT_AUTHORITY_INVALID
(anonymous) @ vendor-shared.js:249377
request @ vendor-shared.js:95315
sendRequest @ vendor-shared.js:697362
sendPayloads @ vendor-shared.js:697338
(anonymous) @ vendor-shared.js:697181
sentryWrapped @ vendor-shared.js:248801
vendor-shared.js:95308 Uncaught (in promise) {error: TypeError: Failed to fetch, status: 1}
(anonymous) @ vendor-shared.js:95308
Promise.catch (async)
(anonymous) @ vendor-shared.js:95305
wrap @ vendor-shared.js:95278
request @ vendor-shared.js:95315
sendRequest @ vendor-shared.js:697362
sendPayloads @ vendor-shared.js:697338
(anonymous) @ vendor-shared.js:697181
sentryWrapped @ vendor-shared.js:248801

**Comment by alibaghernejad** [comment]
For me, this occurred on a Ubuntu machine (20.04 LTS). After a few days, can't log in to my account yet.

- My Amchine Props: Linux ali-GE620DX 5.4.0-42-generic #46-Ubuntu SMP Fri Jul 10 00:24:02 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
- Postman Version: v7.29.1

Here is **rendere-auth.log** file contents:
`[17938][1596613394362][auth][info]["EventBus~initialize - Success"]
[17938][1596613413915][auth][info]["Authentication~did-fail-load: Error with code ",-101,"https://identity.getpostman.com"]
[17938][1596613413916][auth][warn]["Authentication~Window Failed to load",{"errorCode":-101,"errorDescription":"ERR_CONNECTION_RESET","isTrusted":false,"isMainFrame":true}]
[17938][1596613414045][auth][info]["Authentication~did-navigate:","file://"]
[17938][1596613492769][auth][info]["Authentication~will-navigate:","https://identity.getpostman.com"]
[17938][1596613493254][auth][info]["Authentication~will-navigate:","https://identity.getpostman.com"]
[17938][1596613512370][auth][info]["Authentication~did-fail-load: Error with code ",-101,"https://identity.getpostman.com"]
[17938][1596613512370][auth][warn]["Authentication~Window Failed to load",{"errorCode":-101,"errorDescription":"ERR_CONNECTION_RESET","isTrusted":false,"isMainFrame":true}]
[17938][1596613512499][auth][info]["Authentication~did-navigate:","file://"] `

**Comment by tnstanislavko** [comment]
Postman desktop on Windows 10.
I launch Postman and see you are offline message.
I may sign out and press sign in. Then message "Something went wrong! We're having some difficulties connecting to our authentication service. Try again after some time." appears with "Retry" button.
I tried to change DNS, use the 2nd intetnet provider, reinstall Postman.
I may sign in in web-Postman in a browser.
GET/POST to https://identity.getpostman.com from java program works fine.

logs/main.log:
```[3080][1644338725897][main][info]["Booting Postman 9.12.2, win32-10.0.19044 on x64"]
[3080][1644338725899][main][info]["EventBus~initialize - Success"]
[3080][1644338725901][main][info]["Proxy configuration has not been setup"]
[3080][1644338725901][main][info]["CloudProxyManager~init - Success"]
[3080][1644338725902][main][info]["UpdateHandler~init - Success"]
[3080][1644338726266][main][info]["RuntimeIPCAgent~started: Success"]
[3080][1644338726267][main][info]["LeaderSelection: Initialized successfully"]
[3080][1644338726296][main][info]["Bootstrap-models~bootstrap - Success"]
[3080][1644338728526][main][info]["UpdateHandler~app-update-events - Received event",{"name":"checkForElectronVersionUpdated","namespace":"appUpdate"}]
[3080][1644338728539][main][info]["windowManager: First requester booted successfully."]
[3080][1644338728539][main][info]["windowManager~restoreWindows: Restoring requester windows"]
[3080][1644338728540][main][info]["Main~AppEvents - Received booted event for process requester"]
[3080][1644338730619][main][info]["Main - model-events~user - (users, add) received"]
[3080][1644338731041][main][info]["ProtocolHandler~init - Success with status: true]"]
```

logs/renderer-requester.log:
```[3080][1644338728548][requester][info]["SharedRuntimeListeners~boot - Success"]
[3080][1644338728549][requester][info]["booted - requester"]
[3080][1644338728654][requester][info]["NavigationService ~ [BROWSER] Transitioning io the initial route with the URL:","workspace/29ba1c6f-43ab-4e23-8a6c-27c39a57a069"]
[3080][1644338728686][requester][error]["IPCNode~createClient~something wrong connecting to the IPC server",{"errno":-4058,"code":"ENOENT","syscall":"connect","address":"\\\\.\\pipe\\proxy_node_process_prod-sock","name":"Error","message":"connect ENOENT \\\\.\\pipe\\proxy_node_process_prod-sock","stack":"Error: connect ENOENT \\\\.\\pipe\\proxy_node_process_prod-sock\n    at PipeConnectWrap.afterConnect [as oncomplete] (net.js:1146:16)"}]
[3080][1644338728699][requester][info]["CloudProxyHandler: Proxy configuration has not been setup"]
[3080][1644338728727][requester][error]["EditorService~focus: Could not find route to navigate"]
[3080][1644338730012][requester][info]["SessionValidation - Registering listener"]
[3080][1644338730147][requester][info]["UI pulled the session locked information on init"]
```

logs/renderer-auth.log:
```[3080][1644338731035][auth][info]["EventBus~initialize - Success"]
[3080][1644338731615][auth][info]["Authentication~did-fail-load: Error with code ",-101,"https://identity.getpostman.com"]
[3080][1644338731615][auth][warn]["Authentication~Window Failed to load",{"errorCode":-101,"errorDescription":"ERR_CONNECTION_RESET","isTrusted":false,"isMainFrame":true}]
[3080][1644338731795][auth][info]["Authentication~did-navigate:","file://"]
```

**Comment by ArjunSingh-PM** [comment]
@tnstanislavko Are you still experiencing this issue? If not, could you share if you did something specific to unblock yourself?

**Comment by tnstanislavko** [comment]
@ArjunSingh-PM hello, i reinstalled Windows 10. I had problems with Opera, Firefox, Upwork client and Postman. After reinstallation everything goes ok.

**Comment by baxishrey** [comment]
Hello,

I am using Postman behind corporate VPN.
I am able to login to Postman in browser.

For Postman Desktop app, I am facing issues in login.
When I signin, it takes me to the browser, where I login.
When I am redirected back to Postman app, the UI still shows as not logged in. 

**logs/main.log:**
```
[26552][1690263978286][main][info]["Booting Postman 10.16.0, win32-10.0.19044 on x64"]
[26552][1690263978288][main][info]["EventBus~initialize - Success"]
[26552][1690263978289][main][info]["Proxy configuration has not been setup"]
[26552][1690263978289][main][info]["CloudProxyManager~init - Success"]
[26552][1690263978319][main][info]["performShellDataMigration: Migration has already been done. Bailing out"]
[26552][1690263978320][main][info]["UpdateHandler~init - Success"]
[26552][1690263978321][main][info]["LeaderSelection: Initialized successfully"]
[26552][1690263978322][main][info]["VersionDependencyService: Initialized successfully"]
[26552][1690263978381][main][info]["GPU detected VID 4318 DID 8118 ACTIVE false"]
[26552][1690263978381][main][info]["GPU detected VID 32902 DID 18086 ACTIVE false"]
[26552][1690263978381][main][info]["GPU detected VID 5140 DID 140 ACTIVE false"]
[26552][1690263980430][main][info]["shellController~init: Success"]
[26552][1690263980451][main][info]["shellMeta~init: Success"]
[26552][1690263980451][main][info]["partitionMigrationService: State of V7 partition:"]
[26552][1690263980452][main][info]["partitionMigrationService: V7 partitions",[]]
[26552][1690263980452][main][info]["partitionMigrationService~rectifyNamespaceOfScratchPadPartition: There is no partition with scratchpad namespace"]
[26552][1690263980453][main][info]["partitionMigrationService~rectifyNamespaceOfScratchPadPartition: There is no transit partition which could have been used for scratchpad"]
[26552][1690263980453][main][info]["partitionMigrationService~init: Success"]
[26552][1690263981487][main][info]["partitionMigrationService~runV8PartitionsIntegrity: Two way migration~ removing the following partitions from v8",[]]
[26552][1690263981515][main][info]["userPartitionService~_rectifyTheActivePartitionOfApplication: Setting the active partition to ",null]
[26552][1690263981554][main][info]["userPartitionService~init: Success"]
[26552][1690263981571][main][info]["userPartitionService: State of v8 partition system:"]
[26552][1690263981578][main][info]["shell~init~getPartitionToLoad: loading scratchpad partition"]
[26552][1690263981592][main][info]["userPartitionService: List of partitions:",[{"partitionId":"default","namespace":"users","userId":0,"teamId":0,"lastUpdated":"2023-07-21T13:38:12.860Z"},{"partitionId":"11fe24dc-5629-45d7-9f40-aaa117def011","namespace":"scratchPad","userId":0,"teamId":0,"isDirty":false}]]
[26552][1690263981593][main][info]["userPartitionService: users global meta:",{}]
[26552][1690263985445][main][info]["RuntimeIPCAgent~started: Success"]
[26552][1690263985669][main][info]["UpdateHandler~app-update-events - Received event",{"name":"checkForElectronVersionUpdated","namespace":"appUpdate"}]
[26552][1690263985686][main][info]["windowManager: First requester booted successfully."]
[26552][1690263985686][main][info]["windowManager~restoreWindows: Restoring requester windows"]
[26552][1690263985687][main][info]["Main~AppEvents - Received booted event for process requester.Version undefined UI Version: undefined Desktop Platform Version: 10.16.0"]
[26552][1690264018875][main][info]["VersionDependencyService: Resetting UI version as user logged out/switched account"]
[26552][1690264018876][main][info]["Main - model-events~user - (users, add) received"]
[26552][1690264019447][main][info]["ProtocolHandler~init - Success with status: true]"]
[26552][1690264029547][main][info]["userPartitionService: State of v8 partition system:"]
[26552][1690264029569][main][info]["shell~init~getPartitionToLoad: loading scratchpad partition"]
[26552][1690264029579][main][info]["userPartitionService: List of partitions:",[{"partitionId":"default","namespace":"users","userId":0,"teamId":0,"lastUpdated":"2023-07-21T13:38:12.860Z"},{"partitionId":"11fe24dc-5629-45d7-9f40-aaa117def011","namespace":"scratchPad","userId":0,"teamId":0,"isDirty":false}]]
[26552][1690264029580][main][info]["userPartitionService: users global meta:",{}]
[26552][1690264029623][main][warn]["WindowManager~newRequesterWindow - Bailing requester window creation as first requester is not booted."]
[26552][1690264029624][main][info]["ProtocolHandler~handleOpenUrl: postman://auth/callback?code=c0a8c42679b4045c3558f414d5f9f379c78a79a33d9cb65ec59cd0f4e5874389"]
[26552][1690264029624][main][info]["AuthHandler~handleAuthUrl: Received auth url - postman://auth/callback?code=c0a8c42679b4045c3558f414d5f9f379c78a79a33d9cb65ec59cd0f4e5874389"]
[26552][1690264029624][main][info]["AuthHandler~handleAuthorizationRedirectUrl: Received authorization redirect url - postman://auth/callback?code=c0a8c42679b4045c3558f414d5f9f379c78a79a33d9cb65ec59cd0f4e5874389"]
[26552][1690264030180][main][info]["Shell~init~loginUser: associating the user ","5214757"," with transit partition"]
[26552][1690264032136][main][info]["UpdateHandler~app-update-events - Received event",{"name":"checkForElectronVersionUpdated","namespace":"appUpdate"}]
[26552][1690264032144][main][info]["windowManager: First requester booted successfully."]
[26552][1690264032144][main][info]["windowManager~restoreWindows: Restoring requester windows"]
[26552][1690264032144][main][info]["Main~AppEvents - Received booted event for process requester.Version undefined UI Version: undefined Desktop Platform Version: 10.16.0"]
```

**logs/renderer-requester.log**
```
[26552][1690263987488][requester][info]["ScratchpadV2Service: user is on v1"]
[26552][1690264031976][requester][info]["DesktopLocationService ~ init initialized"]
[26552][1690264031978][requester][info]["NavigationService ~ Initialized: Success"]
[26552][1690264031979][requester][info]["initializeNavigation~boot - Success"]
[26552][1690264031982][requester][info]["EventBus~initialize - Success"]
[26552][1690264031982][requester][info]["Messaging~boot - Success"]
[26552][1690264031987][requester][info]["bootCrashReporter~ Success"]
[26552][1690264031988][requester][info]["Telemetry~boot - Success"]
[26552][1690264031989][requester][info]["bootConfigurations~initialize - Success"]
[26552][1690264031992][requester][info]["Settings~boot - Success"]
[26552][1690264031993][requester][info]["Initialize-db~sanitizeInitialMigrationContext: Bailing out"]
[26552][1690264031998][requester][info]["Migrations: Skipping migrations. No migrations to apply."]
[26552][1690264032049][requester][info]["bootstrapModels - Success"]
[26552][1690264032050][requester][info]["WLModels~boot - Success"]
[26552][1690264032053][requester][info]["Session~boot - Success"]
[26552][1690264032055][requester][info]["IndependentServices~boot - Success"]
[26552][1690264032061][requester][info]["AppModels~boot - Success"]
[26552][1690264032076][requester][info]["initializeConfigurationsValues - Success"]
[26552][1690264032083][requester][warn]["RunnerInterface~cleanupRunner: Failed to clean runner:Invalid criteria.\nDetails:\n  Could not use the provided `where` clause.  Could not filter by `status`: Unrecognized modifier (`notIn`) within provided constraint for `status`.\n"]
[26552][1690264032083][requester][info]["CookiesBootHelper~Migration: Bailing out, no migrations to apply."]
[26552][1690264032092][requester][info]["bootIndependentServicesShared: Success"]
[26552][1690264032092][requester][info]["performSingletonJobs: Success"]
[26552][1690264032096][requester][info]["ExtensibleCollections~boot - Success"]
[26552][1690264032102][requester][info]["Sync~boot - Success"]
[26552][1690264032107][requester][info]["PostUpdateBackup~init: Success"]
[26552][1690264032108][requester][info]["Shared~boot - Success"]
[26552][1690264032108][requester][info]["DBWatcher~boot - Success"]
[26552][1690264032115][requester][info]["ThemeManager~boot - Success"]
[26552][1690264032116][requester][info]["WorkbenchService~boot - Success"]
[26552][1690264032116][requester][info]["ContextBarRegistry~boot - Success"]
[26552][1690264032116][requester][info]["Store~boot - Success"]
[26552][1690264032117][requester][info]["Shortcuts~boot - Success"]
[26552][1690264032130][requester][info]["NewConsole~boot - Success"]
[26552][1690264032132][requester][info]["APIDevInterface~boot - Success"]
[26552][1690264032133][requester][info]["ScratchpadV2Service: Received settings change",true]
[26552][1690264032136][requester][info]["Attempting to create IPCNode to proxy_node_process"]
[26552][1690264032140][requester][info]["Requester~boot - Success"]
[26552][1690264032142][requester][info]["SharedRuntimeListeners~boot - Success"]
[26552][1690264032142][requester][info]["booted - requester"]
[26552][1690264032222][requester][info]["NavigationService ~ [BROWSER] Transitioning io the initial route with the URL:","workspace/29ba1c6f-43ab-4e23-8a6c-27c39a57a069"]
[26552][1690264032278][requester][info]["CloudProxyHandler: Proxy configuration has not been setup"]
[26552][1690264032279][requester][error]["IPCNode~createClient~something wrong connecting to the IPC server",{"errno":-4058,"code":"ENOENT","syscall":"connect","address":"\\\\.\\pipe\\proxy_node_process_prod-sock","name":"Error","message":"connect ENOENT \\\\.\\pipe\\proxy_node_process_prod-sock","stack":"Error: connect ENOENT \\\\.\\pipe\\proxy_node_process_prod-sock\n    at __node_internal_captureLargerStackTrace (node:internal/errors:464:5)\n    at __node_internal_exceptionWithHostPort (node:internal/errors:642:12)\n    at PipeConnectWrap.afterConnect [as oncomplete] (node:net:1157:16)"}]
[26552][1690264032353][requester][error]["EditorService~focus: Could not find route to navigate"]
[26552][1690264032451][requester][info]["SessionValidation - Registering listener"]
[26552][1690264032456][requester][info]["SwitchUserToOfflineAPIClientModal: SwitchUserToOfflineAPIClientModal mounted"]
[26552][1690264032457][requester][info]["SwitchUserToOfflineAPIClientModal: activePage",""]
[26552][1690264032467][requester][info]["UI pulled the session locked information on init"]
[26552][1690264032505][requester][info]["ScratchpadV2Service: upgradeScratchpadUsersToApiClient called"]
```
**logs/renderer-auth.log**
```
[26552][1690264019381][auth][info]["EventBus~initialize - Success"]
[26552][1690264020383][auth][info]["Authentication~did-navigate:","https://identity.getpostman.com"]
[26552][1690264029627][auth][info]["Authentication~handleAuthorizationResponse: Started handling authorization response.",{"code":"c0a8c42679b4045c3558f414d5f9f379c78a79a33d9cb65ec59cd0f4e5874389"}]
[26552][1690264029630][auth][info]["Authentication~handleAuthorizationResponse: Successfully executed function to handle authorization response."]
[26552][1690264030128][auth][info]["Authentication~will-navigate:","https://erisedstraehruoytubecafruoytonwohsi.chromiumapp.org"]

```

**Comment by mjwsfg** [comment]
I get the same specific error as baxishrey, namely in the renderer-requester.log

[6280][1691087735992][requester][error]["IPCNode~createClient~something wrong connecting to the IPC server",{"errno":-4058,"code":"ENOENT","syscall":"connect","address":"\\\\.\\pipe\\proxy_node_process_prod-sock","name":"Error","message":"connect ENOENT \\\\.\\pipe\\proxy_node_process_prod-sock","stack":"Error: connect ENOENT \\\\.\\pipe\\proxy_node_process_prod-sock\n    at __node_internal_captureLargerStackTrace (node:internal/errors:464:5)\n    at __node_internal_exceptionWithHostPort (node:internal/errors:642:12)\n    at PipeConnectWrap.afterConnect [as oncomplete] (node:net:1157:16)"}]
[6280][1691087736048][requester][info]["CloudProxyHandler: Proxy configuration has not been setup"]
[6280][1691087736290][requester][info]["SessionValidation - Registering listener"]
[6280][1691087736300][requester][info]["SwitchUserToOfflineAPIClientModal: SwitchUserToOfflineAPIClientModal mounted"]
[6280][1691087736300][requester][info]["SwitchUserToOfflineAPIClientModal: activePage",""]
[6280][1691087736370][requester][error]["EditorService~focus: Could not find route to navigate"]
[6280][1691087736386][requester][info]["UI pulled the session locked information on init"]

What is not clear and thus makes it hard to track down is it doesn't detail the exact destination that might be getting blocked.

**Comment by zhuyanrobert** [comment]
works fine on web browser，but can't login on desktop.
Win10 OS

logs/renderer-auth.log
[15240][1711093289132][auth][info]["Authentication~did-fail-load: Error with code ",-201,"https://identity.getpostman.com"]
[15240][1711093289133][auth][warn]["Authentication~Window Failed to load",{"errorCode":-201,"errorDescription":"ERR_CERT_DATE_INVALID","isTrusted":false,"isMainFrame":true}]
[15240][1711093289250][auth][info]["Authentication~did-navigate:","file://"]

**Comment by mjwsfg** [comment]
Another follow-up comment, still occurring for multiple people in our environment.  Our "fix" is to uninstall, re-install version 9.31.0, sign-in and then work.  You can let that upgrade and you are "usually" OK for a period of time before it asks to re-authenticate at which point it usually fails again and you start all over again.  Really annoying as there are variables and some other settings that don't sync consistently to the cloud or in this process get reset forcing more reconfig.

**Comment by ArjunSingh-PM** [comment]
This issue is fixed.
